tests/test_utilities.py

```diff
@@ -572,180 +561,6 @@ def test_send_email_notification_message_with_site_defined_values(
                 assert getattr(mail.outbox[0], field) == val
             assert mail.outbox[0].connection is conn
 
-    @ddt.data(
```

Comment:
I am removing this test since I feel like it's no longer necessary to test against a template that is no longer in file form in git, we are doing templates in the db, so whoever changes those templates will need to test them out against the version of code being tested. It would be hard to keep the test and the database templates in sync, and we are going to go Braze so I don't think losing this test case for this time, is going to lose us anything.

If you feel strongly we need this test back, I am going to have to extra work to bring the template into the factory somehow, I used simpler templates in my factory to only cover some essential context variables